tests/GLPITestCase.php

```diff
@@ -52,7 +52,6 @@ public function setUp() {
    }
 
    public function beforeTestMethod($method) {
-      unset($_SESSION['glpicronuserrunning']);
```

Comment:
`$this->array($_SESSION)->notHasKey('glpicronuserrunning')`

Reply:
Just saw the `GLPITestCase::setUp()`, I have to take another look at all of this.

Reply:
I added session reset in `beforeTestMethod`.  So session should only contains `glpi_currenttime`, `glpi_use_mode`, and all the default preferences (to get same behavior as what is done during global includes by config.php script).